package.json

```diff
@@ -106,7 +106,17 @@
   "commitlint": {
     "extends": [
       "@commitlint/config-conventional"
-    ]
+    ],
+    "rules": {
+      "scope-case": [
+        2,
+        "always",
+        [
+          "lower-case",
+          "pascal-case"
```

Comment:
Added pascal case so we can do things like `feat(PeopleAdapter):`

Reply:
Should we be specifying the scope as a filename? Since we didn't have this before, I was thinking it was actually a little bit more useful to specify the "area" where the change was made, for instance, for this PR it would be `feat(people)`.
For my PR I did `feat(activity)`

Reply:
I like that the scope translates to a specific thing that users will import. It makes the change log align with what users are familiar with, like:

### Features

* **WebexAvatar:** add support for animated profiles like Harry Potter